vignettes/getting-osm-data.Rmd

```diff
@@ -65,13 +65,12 @@ bucharest <- list(
   railways = railways
 )
 
-walk(
-  st_write_list,
```

Comment:
When updating the name of the dataset, I have noticed this function probably had a placeholder for the name of the dataset (`st_write_list` -> `bucharest`)? Also I made a suggestion to replace `walk` for `walk2`, which  seems to simplify quite a bit the overall snippet?

Reply:
yes, much better!